src/Shortcuts/ShortcutItem.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2015 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React, { FC } from 'react';
+import Card from 'react-bootstrap/Card';
+import { string } from 'prop-types';
+
+import './Shortcut-item.scss';
+
+export interface Props {
+    title: string;
+    hotKey: string;
+    description: string;
+}
+
+const ShortcutItem: FC<Props> = ({ title, hotKey, description }) => {
+    const shortcutKeys: string[] = hotKey.split('+');
+    return (
+        <Card className="shortcut-item">
+            <Card.Body>
+                <Card.Title>{title}</Card.Title>
+                <Card.Text>
+                    {shortcutKeys.map(shortcutKey => (
+                        <span key={shortcutKey}>
+                            <span className="shortcut-span">{shortcutKey}</span>
+                            {shortcutKey !==
```

Comment:
Alternatively use something like the index to check for last item, and use a short-circuiting && instead of a ternary.

Reply:
E.g. { a === b ? c : null } => { a === b && c }